src/components/EditTrip.js

```diff
@@ -0,0 +1,17 @@
+import { useHistory } from "react-router";
+
+export default function EditTrip() {
+  const history = useHistory();
+  return (
+    <div className="editTrip">
+      <h2>Edit Page</h2>
+      <button
+        onClick={(e) => {
```

Comment:
You are not using the `e` parameter here. You could just remove it (probably eslint is telling you that as well if you check)